apps/server/src/modules/authentication/uc/account.uc.ts

```diff
@@ -174,13 +176,15 @@ export class AccountUc {
 	 * @param params account details
 	 */
 	async updateMyAccount(currentUserId: EntityId, params: PatchMyAccountParams) {
-		let account: Account;
+		let user: User;
 		try {
-			account = await this.accountRepo.findByUserId(currentUserId);
+			user = await this.userRepo.findById(currentUserId, true);
 		} catch (err) {
-			throw new EntityNotFoundError('Account');
+			throw new EntityNotFoundError('User');
```

Comment:
Do we need try/catch or will the repo automatically throw an 'EntityNotFoundError'? On problem is, that every error from the repository will be treated as not found, but it can also be something like can't connect to database which should results in an internal server error and not not found.

Reply:
Done